inst/rmarkdown/templates/nhsr-presentation/skeleton/skeleton.Rmd

```diff
@@ -5,7 +5,22 @@ author: "Author Name"
 institute: "NHS Organisation"
 date: "Presentation Date"
 output:
-  nhsrtheme::nhsr_presentation: default
+  xaringan::moon_reader:
+    css: 
+      - default
+      - css/nhsr.css
+      - css/nhsr-fonts.css
+    lib_dir: libs
+    seal: false
```

Comment:
because of the type of title slide, I think it doesn't make sense to ever have seal: true for this package, so I throw an error in nhsr_presentation() if this parameter is set

Reply:
I think that makes perfect sense 👍